cmd/dex/config.go

```diff
@@ -6,9 +6,9 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/sirupsen/logrus"
 	"golang.org/x/crypto/bcrypt"
 
+	"github.com/dexidp/dex/pkg/log"
```

Comment:
nit: can this be `internal/log` instead of `pkg/log`? :)

Reply:
See my comment bellow.